components/src/core/Drawer/DrawerFooter.tsx

```diff
@@ -1,78 +0,0 @@
-import React, { HTMLAttributes } from 'react';
```

Comment:
Just to make sure that we haven't accidentally introduced a breaking change here related to the folder restructuring, did we check to make sure that all of these components that were relocated can still be individually imported from their original (prior) location?

e.g.:

```tsx
import { DrawerFooter } from '@brightlayer-ui/react-components/core/Drawer/DrawerFooter';
```

Reply:
this works out just fine!